oak/common/label.h

```diff
@@ -47,10 +47,14 @@ oak::label::Label DeserializeLabel(const std::string& label_bytes);
 // provided authorization bearer token.
 oak::label::Label AuthorizationBearerTokenLabel(const std::string& authorization_token_hmac);
 
-// Creates a label that only allows declassifying data for modules that match the
-// provided module attestation.
+// Creates a label that only allows declassifying data for modules that match the provided module
+// attestation.
```

Comment:
pre-existing: the form of the "module attestation" is vague &ndash; deliberately so?  Or can we now be more specific (", in the form of a SHA-256 hash of the Web Assembly module")?

Reply:
Good point, I'll do it in #1070 since I'm at it.